sqlglot/expressions.py

```diff
@@ -6578,6 +6582,28 @@ def rename_table(old_name: str | Table, new_name: str | Table) -> AlterTable:
     )
 
 
+def rename_column(
+    table_name: str | Table, old_column: str | Column, new_column: str | Column, exists: Boolean
+) -> AlterTable:
+    """Build ALTER TABLE... RENAME COLUMN... expression
+
+    Args:
+        table_name: Name of the table
+        old_column: The old name of the column
+        new_column: The new name of the column
+
+    Returns:
+        Alter table expression
+    """
+    table = to_table(table_name)
+    return AlterTable(
+        this=table,
+        actions=[
+            RenameColumn(this=old_column, to=new_column, exists=exists),
```

Comment:
This isn't quite right, you need to convert possible strings into `Column`s (same as what you did on L6598):
```suggestion
            RenameColumn(this=to_column(old_column), to=to_column(new_column), exists=exists),
```

Reply:
updated to use old_column_name and new_column_name in args, similar to table_name